package.json

```diff
@@ -127,7 +127,7 @@
     "react-loadable": "5.5.0",
     "react-plaid-link": "3.2.1",
     "react-redux": "7.0.2",
-    "react-router-dom": "5.2.0",
+    "react-router-dom": "^5.2.0",
```

Comment:
Since we already had `react-router-dom` installed, this added the `^` to our package.json. We're keeping our packages locked at their current versions for right now, so you'll need to remove the `^`.